generics/helpers/utils.js

```diff
@@ -175,17 +175,17 @@ function valueParser(dataToBeParsed) {
  */
 
 function filterLocationIdandCode(dataArray) {
-  let entityId = [];
+  let locationIds = [];
```

Comment:
@praveenKDass we might not need this changes hence we have to use entity management service

Reply:
Will check that in entity-management PR review @VISHNUDAS-tunerlabs